Tests/GRPCTests/GRPCAsyncServerHandlerTests.swift

```diff
@@ -213,28 +236,22 @@ class AsyncServerHandlerTests: ServerHandlerTestCaseBase {
     // Wait for tasks to finish.
     await handler.userHandlerTask?.value
 
-    await assertThat(self.recorder.metadata, .is(.nil()))
+    await assertThat(self.recorder.metadata, .nil())
     await assertThat(self.recorder.messages, .isEmpty())
     await assertThat(self.recorder.status, .notNil(.hasCode(.internalError)))
   } }
 
-  // TODO: Running this 1000 times shows up a segfault in NIO event loop group.
```

Comment:
@glbrntt FYI, I think whatever we did in refactoring and/or the use of a more up to date toolchain means I can no longer reproduce this behaviour.